src/main.ts

```diff
@@ -83,6 +84,11 @@ async function run(): Promise<void> {
           }
         }
 
+        foundTestToolPath = path.join(
```

Comment:
Probably should verify this actually exists. Just because VS is installed doesn't mean the test runner is.